src/main/java/com/google/sps/servlets/UserLoginServlet.java

```diff
@@ -34,11 +37,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String loggedOutUrl = userService.createLogoutURL("/index.html");
       loginInfo.addProperty("LogOutUrl", loggedOutUrl);
       loginInfo.addProperty("LogInUrl", "");
-
+      try {
+        if(SessionContext.getInstance() != null){
+            boolean value = SessionContext.getInstance().isUserExistingInDatastore();
```

Comment:
Please rename the value variable to a more descriptive name (L42).

Reply:
Done